app/src/main/java/com/example/recipeapp/fragments/ProfileFragment.java

```diff
@@ -95,7 +115,109 @@ public void done(List<Recipe> objects, ParseException e) {
         });
     }
 
+
     private void setProfileImage() {
-        Glide.with(getContext()).load(CURRENT_USER.getProfileImage().getUrl()).circleCrop().into(binding.ivProfileImage);
+        CURRENT_USER.setProfileImage(new ParseFile(photoFile));
+        CURRENT_USER.getParseUser().saveInBackground(new SaveCallback() {
+            @Override
+            public void done(ParseException e) {
+                if (e != null) {
+                    Log.e(TAG, "Issue with saving profile image!", e);
+                    Toast.makeText(getContext(), "Unable to save profile image. Please try again!", Toast.LENGTH_SHORT).show();
+                    return;
+                } else {
```

Comment:
nit: probably don't need the else clause here since the above condition `returns`